### PR TITLE
remove old couch user models

### PR DESCRIPTION
**Pull request by snopoke**
@czue @dannyroberts 

cc @NoahCarnahan

### CODE REVIEW

File: corehq/apps/users/models.py [hunk 1]
```diff
@@ -1142,20 +1142,11 @@ def from_old_couch_user(cls, old_couch_user, copy_id=True):
 
     @classmethod
     def wrap_correctly(cls, source):
-        if source['doc_type'] == 'CouchUser' and \
-                source.has_key('commcare_accounts') and \
-                source.has_key('web_accounts'):
-            from . import old_couch_user_models
-            # todo: remove this functionality and the old user models module
-            logging.error('still accessing old user models')
```
Comment (snopoke) [inline corehq/apps/users/models.py]: I searched couch log and emails for this and nothing came up

Comment (czue) [inline corehq/apps/users/models.py]: how'd you search? I noticed that our search box on prod doesn't work anymore

Comment (snopoke) [inline corehq/apps/users/models.py]: Well that's not very useful! Running the search directly still returned no
results (at least none containing the search string)

**Comment by dannyroberts** [comment]
:smile: Bout time!

**Pull request merged**